base-notebook/Dockerfile

```diff
@@ -79,6 +73,13 @@ RUN cd /tmp && \
     fix-permissions $CONDA_DIR && \
     fix-permissions /home/$NB_USER
 
+# Install Tini
+RUN conda install --quiet --yes 'tini=0.18.0' && \
+    conda list tini | grep tini | tr -s ' ' | cut -d ' ' -f 1,2 >> $CONDA_DIR/conda-meta/pinned && \
+    conda clean -tipsy && \
+    fix-permissions $CONDA_DIR && \
+    fix-permissions /home/$NB_USER
```

Comment:
Do we need to fix permissions on the user home directory? Or just `$CONDA_DIR`?

Reply:
Not totally sure. `conda` does create things like `~/.conda`. I don't think it's contents would have changed here, but it seemed ok to keep this to be safe.